main.go

```diff
@@ -164,7 +180,11 @@ func messageStatus(event *corev2.Event) string {
 	case 0:
 		return "Resolved"
 	case 2:
-		return "Critical"
+		if config.slackAlertCritical {
+			return "<!channel> Critical"
```

Comment:
Perhaps I'm missing something, but shouldn't this be `@channel Critical`? Or does this somehow get translated into an `@`?